internal/server/statvar/statvar_group.go

```diff
@@ -155,16 +155,21 @@ func GetStatVarGroup(
 		if err != nil {
 			return nil, err
 		}
+		// btDataList is ranked by pereference. Pick the svg Node information
+		// from the most preferred import group.
 		for _, btData := range btDataList {
 			for _, row := range btData {
-				svg, ok := row.Data.(*pb.StatVarGroups)
-				if ok {
-					result = svg
+				if svg, ok := row.Data.(*pb.StatVarGroups); ok {
+					for dcid := range svg.StatVarGroups {
+						if _, ok := result.StatVarGroups[dcid]; !ok {
+							result.StatVarGroups[dcid] = svg.StatVarGroups[dcid]
+						}
+					}
```

Comment:
To understand, this can pick some SVGs from certain import-groups and some other SVGs from other import-groups?  I kind of worry it can cause some dangling links, etc...

As such, we can ensure custom import-group includes all the base SVs + custom SVs, so is it sufficient if we prefer custom first?  (Sorry if thats how things are and this was unnecessary churn)